src/css/components/menu/_menu.scss

```diff
@@ -43,13 +43,13 @@
 }
 
 .vjs-menu li.vjs-menu-item:focus,
-.vjs-menu li.vjs-menu-item:hover {
+.vjs-menu .vjs-menu-content li.vjs-menu-item:hover {
```

Comment:
why increase the specificity?

Reply:
The CSS ".js-focus-visible .video-js *:focus:not(.focus-visible)" selector has higher prioryty than ".vjs-menu li.vjs-menu-item:hover". So if the first menu item gets focus, menu item doesn't change background color on hover. That's way I increased the specificity to the hover CSS selector to resolve above problem.

Reply:
Ah, hm... I'd be worried that increasing this specificity would break some custom skins

Reply:
@gkatsev Does it look better now?